examples/calibration/plot_compare_calibration.py

```diff
@@ -86,17 +92,23 @@ def predict_proba(self, X):
 
 from sklearn.calibration import CalibrationDisplay
 from sklearn.ensemble import RandomForestClassifier
-from sklearn.linear_model import LogisticRegression
+from sklearn.linear_model import LogisticRegressionCV
 from sklearn.naive_bayes import GaussianNB
 
-# Create classifiers
-lr = LogisticRegression()
+# Create classifiers: here we tune the main hyper-parameter automatically for
+# the logistic regression model because its efficient and concise to do so.
```

Comment:
This sentence doesn't work out

Reply:
I reworked it in the subsequent commits.